inventory/templates/inventory/catalogue_for_type.html

```diff
@@ -90,4 +117,10 @@ <h1>Item catalogue for {{ item_type }}</h1>
 {% block js_bottom %}
     {{ block.super }}
     <script src="{% static "js/enlarge_item_image.js" %}"></script>
+
+    <script language="JavaScript">
```

Comment:
Might as well make this a snippet in a separate template; it's being used in activity_page_base.html as well and I easily see it being used elsewhere in the future too.

Reply:
Die gebruikte popover, maar ik heb beide naar utils verplaatst nu als js-script.